src/components/offline-icon.tsx

```diff
@@ -1,14 +1,10 @@
 export default (
 	<svg viewBox="0 0 16 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-		<path
-			d="M14.3616 11.2468C15.339 10.682 15.9996 9.60352 15.9996 8.36597C15.9996 6.73087 14.8465 5.37347 13.3331 5.11157L13.3333 5.06597C13.3333 2.63597 11.1843 0.666016 8.5333 0.666016C7.2983 0.666016 6.17225 1.09355 5.32167 1.79598L6.36271 2.88434C6.9296 2.44576 7.67855 2.16602 8.5333 2.16602C10.4814 2.16602 11.8333 3.58458 11.8333 5.06597L11.8332 5.09497L11.8191 6.37187L13.0773 6.58967C13.8512 6.72357 14.4996 7.43967 14.4996 8.36597C14.4996 9.22043 13.9545 9.88797 13.262 10.0972L14.3616 11.2468Z"
-			fill="white"
-		/>
+		<path d="M14.3616 11.2468C15.339 10.682 15.9996 9.60352 15.9996 8.36597C15.9996 6.73087 14.8465 5.37347 13.3331 5.11157L13.3333 5.06597C13.3333 2.63597 11.1843 0.666016 8.5333 0.666016C7.2983 0.666016 6.17225 1.09355 5.32167 1.79598L6.36271 2.88434C6.9296 2.44576 7.67855 2.16602 8.5333 2.16602C10.4814 2.16602 11.8333 3.58458 11.8333 5.06597L11.8332 5.09497L11.8191 6.37187L13.0773 6.58967C13.8512 6.72357 14.4996 7.43967 14.4996 8.36597C14.4996 9.22043 13.9545 9.88797 13.262 10.0972L14.3616 11.2468Z" />
 		<path
 			fillRule="evenodd"
 			clipRule="evenodd"
 			d="M3.75555 11.6659H10.6867L12.458 13.5177C12.7443 13.817 13.2191 13.8276 13.5184 13.5413C13.8177 13.2549 13.8283 12.7802 13.542 12.4809L2.54198 0.980852C2.25567 0.681524 1.78091 0.670974 1.48158 0.957287C1.18226 1.2436 1.17171 1.71836 1.45802 2.01769L3.34162 3.9869C1.46387 4.18881 0 5.82609 0 7.81597C0 9.94227 1.67147 11.666 3.73333 11.666L3.75555 11.6659ZM4.79619 5.50759L3.82248 5.46777C3.793 5.46657 3.76328 5.46597 3.73333 5.46597C2.54307 5.46597 1.5 6.47427 1.5 7.81597C1.5 9.15625 2.54088 10.1638 3.72957 10.166L3.75105 10.1659H9.25193L4.79619 5.50759Z"
-			fill="white"
```

Comment:
Not sure if this is the good approach. I needed to be able to set a different fill on the offline icon but I could not override it with className since it is set here. Removing it here does not break the tooltip and icon implementation since we are setting the fill directly there: https://github.com/Automattic/studio/blob/e428f51736604a1ca477b21c546932fddb9eb8be/src/components/tooltip.tsx#L54

However, there might be a better approach

Reply:
I think this is a good call. Another approach would be to replace the `fill` color here with `currentColor`. This value will make the icon match the text color where it's used. This will allow us to remove the `fill` class name from [here](https://github.com/Automattic/studio/pull/271/files#diff-d3d6432d7d681344c73a14e2e30339dbdd7fe784d4446baf42efaf59eed40b33R229).